docs/tool_specs/eof/eof_specs.md

```diff
@@ -0,0 +1,35 @@
+
+
+# Ethereum EVM Object Format code verification tool
+
+Each client is required to implement the `evm eof` tool, which is used to validate eof code tests by verifying eof container validity.
+
+**Format**: `[eof code] -> [result]`
+
+## EVM EOF CLI:
+
+**Usage:**
+
+    evm-eof [command options] [arguments...]
+    echo $inputstream | evm-eof
+
+**Options:**
+
+```
+Parameters:
+  --state.fork value (required)
+	   Sets the fork rules and all EIPs included in it for eof code validation.
+	   Current supported forks and EIPs: ["Prague"]
+
+Inputs:
+  --hex code (required)
+	   Direct input of the evm eof bytecode
+
+
+Outputs:
```

Comment:
This doesn't vibe with the tools that were written for shanghai - https://github.com/holiman/txparse/blob/main/README.md#eof-parser-eofparse - Each client already had one thing set up for this

Key changes needed::
* support a `stdin` mode where there are multiple inputs (one per line) and one output per input.
  * This is to counteract the startup penalty that half the clients have.
* Success has an output, not just terminating the validator
  * The output includes the parsed code sections.  This is usefull in detecting improper parsing on success.
  * Hencee, the success output should be required to match